test/suites/smoke-test-common-all/test-inactivity-tracking.ts

```diff
@@ -0,0 +1,91 @@
+import "@tanssi/api-augment/dancelight";
+
+import { beforeAll, describeSuite, expect } from "@moonwall/cli";
+import type { ApiPromise } from "@polkadot/api";
+
+describeSuite({
+    id: "S09",
+    title: "Inactivity tracking suit",
+    foundationMethods: "read_only",
+    testCases: ({ it, context, log }) => {
+        let api: ApiPromise;
+        let lastSessionIndex: number;
+        let lastSessionEndBlock: number;
+        let chain: any;
+
+        beforeAll(async () => {
+            api = context.polkadotJs();
+            chain = api.consts.system.version.specName.toString();
+            lastSessionIndex = (await api.query.session.currentIndex()).toNumber() - 1;
+
```

Comment:
we probably want to extract this function to `utils/block.ts`

Reply:
Yup updated in [e663aaf](https://github.com/moondance-labs/tanssi/pull/1052/commits/e663aaf4b5ae9678c132f3c043fd50900af823a7)